Error while checking build/O2/o2 for 4c839a25a0c5184529a38ae358daf45a83d16eb0 at 2022-09-22 14:26:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/9902-slc7_x86-64/0/Framework/Foundation/include/Framework/Traits.h:47:35: error: pack expansion argument for non-pack parameter 'T' of alias template 'template<class T> using SmallGroups = o2::soa::SmallGroupsBase<T, true>'
/sw/SOURCES/O2/9902-slc7_x86-64/0/Framework/Foundation/include/Framework/Traits.h:47:35: error: pack expansion argument for non-pack parameter 'T' of alias template 'template<class T> using SmallGroups = o2::soa::SmallGroupsBase<T, true>'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9902/4c839a25a0c5184529a38ae358daf45a83d16eb0/build_O2_o2/pretty.html).

--- a/Framework/Core/include/Framework/AnalysisTask.h
+++ b/Framework/Core/include/Framework/AnalysisTask.h
@@ -196,7 +196,7 @@ struct AnalysisDataProcessorBuilder {
       info.selection = framework::expressions::createSelection(table, info.filter);
       info.resetSelection = false;
     }
-    if constexpr (!is_smallgroups_v<std::decay_t<T>>) {
+    if constexpr (!o2::soa::is_smallgroups_v<std::decay_t<T>>) {
       if (info.selection == nullptr) {
         throw runtime_error_f("Null selection for %d (arg %d), missing Filter declaration?", info.processHash, info.argumentIndex);
       }

--- a/Framework/Core/include/Framework/AnalysisTask.h
+++ b/Framework/Core/include/Framework/AnalysisTask.h
@@ -196,7 +196,7 @@ struct AnalysisDataProcessorBuilder {
       info.selection = framework::expressions::createSelection(table, info.filter);
       info.resetSelection = false;
     }
-    if constexpr (!framework::is_base_of_template_v<soa::SmallGroups, std::decay_t<T>>) {
+    if constexpr (!is_smallgroups_v<std::decay_t<T>>) {
       if (info.selection == nullptr) {
         throw runtime_error_f("Null selection for %d (arg %d), missing Filter declaration?", info.processHash, info.argumentIndex);
       }

